app/assets/favicons/manifest.json.erb

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "OpenStreetMap",
+	"name": "OpenHistoricalMap",
 	"icons": [
 		{
 			"src": "<%= image_path("android-chrome-36x36.png").gsub("/", "\\/") %>",
```

Comment:
Looks like we should also update these images to be the OHM logo instead of the OSM logo. Unfortunately, [we don’t have a good logo yet](https://forum.openhistoricalmap.org/t/designing-a-new-openhistoricalmap-logo/62), so we can let this slide for now.